docs/book/component-guide/experiment-trackers/mlflow.md

```diff
@@ -209,6 +209,6 @@ def step_one(
     ...
 ```
 
-Check out the [SDK docs](https://sdkdocs.zenml.io/latest/integration_code_docs/integrations-mlflow.html#zenml.integrations.mlflow) for a full list of available attributes and [this docs page](https://docs.zenml.io/how-to/pipeline-development/use-configuration-files/runtime-configuration) for more information on how to specify settings.
+Check out the [SDK docs](https://sdkdocs.zenml.io/latest/integration_code_docs/integrations-mlflow.html#zenml.integrations.mlflow) for a full list of available attributes and [this docs page](https://docs.zenml.io/concepts/steps_and_pipelines/configuration) for more information on how to specify settings.
```

Comment:
just noticed this, but why are we using underscores here whereas we use hyphens everywhere else for folder names?

Reply:
Seems like it's a recipe for confusion. I'd maybe add it to a list of things for when / if we do the cleanup that he wanted. Probably not something for this PR; just I noticed it while reading..